src/watch_utils.py

```diff
@@ -109,15 +111,14 @@ def create_file_tree(summaries, fs_events):
 Include the above items in your response exactly as is, along all other proposed changes.
 """.strip()
 
-    client = Groq()
-    cmpl = client.chat.completions.create(
+    cmpl = litellm.completion(
         messages=[
             {"content": FILE_PROMPT, "role": "system"},
             {"content": json.dumps(summaries), "role": "user"},
             {"content": WATCH_PROMPT, "role": "system"},
             {"content": json.dumps(fs_events), "role": "user"},
         ],
-        model="llama3-70b-8192",
```

Comment:
I don't like this global variable, because I think it can switch incognito mode on/off for different requests or paths. But I don't want to learn about the watch handler global function whatever, and definitely don't plan on testing it.